src/main/webapp/pages/pipelines/types/generic_pipeline.html

```diff
@@ -207,13 +208,19 @@ <h1 th:text="${mainTitle}"></h1>
                                         </div>
                                     </div>
                                 </div>
+                                <div class="col-sm-10" th:if="${automatedProject != null}">
```

Comment:
Can you move this to above the parameter selection.

Reply:
This is here to replace the checkboxes for sharing results with a project.  I added the message there to show it's auto shared.  I guess I could show the project name with a checked (and disabled) checkbox instead.  What do you think?

Reply:
Sorry don't know what your comment actually means, I will come by your desk to talk to you.  This is what I was referring to:

![Screen Shot 2019-03-11 at 9 47 39 AM](https://user-images.githubusercontent.com/11295750/54132825-e975eb80-43e2-11e9-9fb4-738ab8c96707.png)